metricbeat/module/logstash/_meta/Dockerfile

```diff
@@ -1,5 +1,4 @@
 FROM docker.elastic.co/logstash/logstash:6.6.0
 
-COPY healthcheck.sh /
 ENV XPACK_MONITORING_ENABLED=FALSE
-HEALTHCHECK --interval=1s --retries=300 CMD sh /healthcheck.sh
+HEALTHCHECK --interval=1s --retries=300 CMD curl -f http://localhost:9600/_node/stats
```

Comment:
Seems like the old one was actually the more sophisticated one. If I remember correctly we had it this way also because of some flakyness with LS starting up, so I wonder if we should use the old one everywhere instead?

Reply:
Addressed in 38b8f0b.